origin.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Andre Lau Portfolio</title>
+    <link rel="stylesheet" href="./main.css">
+</head>
+
+<body>
+    <header>
+        <nav class="navbar">
+            <h3>Hector Andre Lau</h3>
+            <a href=""><img src="./images/Union.svg" alt="Menu"></a>
+        </nav>
```

Comment:
- [ ] Please, consider, as was mentioned by a previous reviewer, your logo, which is the "Hector Andre Lau" should have a link (`a` tag)